lib/src/query_component.dart

```diff
@@ -31,9 +33,58 @@ class QueryComponent {
   QueryComponent(this._queryService, this._excel);
 
   void onClick() async {
-    final jsonResponse =
-        await _queryService.execQuery(advertiserId, insertionOrderId);
-    final parsedResponse = InsertionOrderParser.parse(stringify(jsonResponse));
-    await _excel.populate([parsedResponse]);
+    // Uses DV360 public APIs to fetch entity data.
```

Comment:
Overall this looks good to me.

This seems focussed on single IO case. Are there batch apis we can use? What is the plan for advertiser level query?

Reply:
For public dv3 api, we can use [this](https://developers.google.com/display-video/api/reference/rest/v1/advertisers.insertionOrders/list), which lists all the IOs under an advertiser, and we add also add a filter to let it query IOs under the same median plan. 

For reporting api, we can also batch query all the IOs under advertiser. 

Advertiser level query is coming soon!